src/Controller/V2/Document/DocumentRoute.ts

```diff
@@ -159,19 +151,12 @@ export class DocumentRoute extends BaseRoute {
       payload,
       user
     )
-    res.sendStatus(StatusCodes.OK).end()
-    this.sendDataToClients(result, manuscriptID)
-  }
-  private async listen(req: Request, res: Response) {
-    const { manuscriptID, projectID } = req.params
-    const user = req.user
-    const result = await this.documentController.getEvents(projectID, manuscriptID, 0, user)
-    const data = this.formatDataForSSE({ ...result, transformVersion: getVersion() })
-    res.setHeader('Content-Type', 'text/event-stream')
-    res.setHeader('Connection', 'keep-alive')
-    res.setHeader('Cache-Control', 'no-cache')
-    res.write(data)
-    this.manageClientConnection(req, res)
+    if (process.env.NODE_ENV === 'development') {
+      this.documentController.broadcastSteps(manuscriptID, JSON.stringify(result))
+      res.sendStatus(StatusCodes.OK).end()
+    } else {
+      res.status(StatusCodes.OK).send(JSON.stringify(result))
```

Comment:
Is this a response that durable object worker expects?

Reply:
Yeah, the durable object will broadcast the data to its clients. (The data is not yet used by the FE though)

Reply:
https://github.com/atypon/manuscripts-cf-worker/pull/1/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R28-R34

Reply:
got it, thanks